doc/aurpublish.1.asciidoc

```diff
@@ -0,0 +1,83 @@
+aurpublish(1)
+=============
+
+Name
+----
+aurpublish - PKGBUILD management framework for the Arch User Repository
+
+
+Synopsis
+--------
+'aurpublish' [options] PACKAGE
+'aurpublish' log [<options>] PACKAGE
+'aurpublish' setup
+
+
+Description
+-----------
+Aurpublish is an utility which helps maintainers of aur packages.
```

Comment:
"... to submit updates" maybe?

Reply:
Fixed in https://github.com/eli-schwartz/aurpublish/pull/16/commits/03d4d0c704641ca9e9e703aa86dbad81ffb8da51.